docs/content/docs/3.recipes/3.prisma.md

```diff
@@ -0,0 +1,187 @@
+---
+title: Prisma ORM
+description: Learn how to setup Prisma ORM with NuxtHub.
+---
+
+::alert{icon="i-ph-warning" color="red" title="WIP"}
+This ORM is not fully compatible with NuxtHub yet.
+:br
+- [Migrations](#migrations) are not supported yet.
+- Deployment has an addressed issue, that will be solved soon ([unwasm/issues#21](https://github.com/unjs/unwasm/issues/21){target=_blank}).
+::
+
+::callout{icon="i-ph-link" to="https://www.prisma.io/orm" external}
+Learn more about **Prisma ORM**.
+::
+
+## Setup
+
+To enhance your Developer Experience with the database, we can create a `usePrisma()` server composable with few steps.
+
+### Install Prisma
+
+1. Install the `prisma`, `@prisma/adapter-d1` and `@prisma/client` packages to your project:
+
+::code-group
+```bash [pnpm]
+pnpm add prisma @prisma/adapter-d1 @prisma/client
+```
+```bash [yarn]
+yarn add prisma @prisma/adapter-d1 @prisma/client
+```
+```bash [npm]
+npm install prisma @prisma/adapter-d1 @prisma/client
+```
+```bash [bun]
+bun add prisma @prisma/adapter-d1 @prisma/client
+```
+::
+
+### Init Prisma
+
+To init Prisma and generate its schema file, run this command:
+
+```bash
+npx prisma init --datasource-provider sqlite
+```
+
+Now you have `.prisma/schema.prisma` in your project, make it match this config:
+
+```[schema.prisma]
+generator client {
+  provider = "prisma-client-js"
+  // preview feature enabling binding adapter
+  previewFeatures = ["driverAdapters"]
+}
+
+datasource db {
+  provider = "sqlite"
+  // url is required but won't be used
+  url = env("DATABASE_URL")
+}
+```
+
+::note
+No need to define `DATABASE_URL` in your `.env` file, it won't be used.
+::
```

Comment:
````suggestion
### 2. Init Prisma

Add `@prisma/nuxt` to the `modules` section of `nuxt.config.ts`
```javascript
export default defineNuxtConfig({
  modules: ["@prisma/nuxt"],
});

To generate needed directories and files, run this command:

```bash
npx prisma init --datasource-provider sqlite
```

Now you have `.prisma/schema.prisma` in your project, make it match this config:

```[schema.prisma]
generator client {
  provider = "prisma-client-js"
  // preview feature enabling binding adapter
  previewFeatures = ["driverAdapters"]
}

datasource db {
  provider = "sqlite"
  // url is required but won't be used
  url = env("DATABASE_URL")
}
```

::note
No need to define `DATABASE_URL` in your `.env` file, it won't be used.
::
````